statuses.py

```diff
@@ -63,4 +63,6 @@
    "Virtual hugs can be just as good as real hugs.",
    "Be a love pharmacist; dispense hugs like medicine — they are!",
    "A hug in time saves nine 🙂.",
+   "Hey you, i'm talking to you, yes you, i'm sending you a hug",
+   "Roses are red, violets are blue, You're so great, I just want to hug you!",
```

Comment:
For this one I think it would look better if each stanza in the poem were in its own line. You can see the message in line 22 in this file for an example of how this poem was used.

Reply:
Fixed it! I hope you like it.